src/lighteval/models/model_config.py

```diff
@@ -348,9 +348,6 @@ def create_model_config(  # noqa: C901
         if model_args.pop("dummy", False):
             return DummyModelConfig(**model_args)
 
```

Comment:
Why did you remove vllm here?

Reply:
The whole function can be removed I think

Reply:
then you should to remove dedups :)